test_module.py

```diff
@@ -16,25 +16,29 @@ def setUp(self):
             tor_sock_port = 9150
             tor_control_port = 9151
         tor_password = "scholarly_password"
-        scholarly.nav._setup_tor(tor_sock_port, tor_control_port, tor_password)
+        scholarly.use_tor(tor_sock_port, tor_control_port, tor_password)
 
     def test_launch_tor(self):
         """
         Test that we can launch a Tor process
         """
         if sys.platform.startswith("linux"):
-            tor_cmd = '/usr/bin/tor'
+            tor_cmd = 'tor'
         elif sys.platform.startswith("win"):
-            tor_cmd = 'C:\\Tor\\tor.exe'
+            tor_cmd = 'tor.exe'
 
         tor_sock_port = random.randrange(9000, 9500)
```

Comment:
Maybe make the ports fixed? the rand might collide with something and we would get random fails.